schema/gen/go/genUnionReprStringprefix.go

```diff
@@ -0,0 +1,258 @@
+package gengo
+
+import (
+	"io"
+
+	"github.com/ipld/go-ipld-prime/schema"
+	"github.com/ipld/go-ipld-prime/schema/gen/go/mixins"
+)
+
+var _ TypeGenerator = &unionReprStringprefixGenerator{}
+
+func NewUnionReprStringprefixGenerator(pkgName string, typ *schema.TypeUnion, adjCfg *AdjunctCfg) TypeGenerator {
+	return unionReprStringprefixGenerator{
+		unionGenerator{
+			adjCfg,
+			mixins.MapTraits{
+				PkgName:    pkgName,
+				TypeName:   string(typ.Name()),
+				TypeSymbol: adjCfg.TypeSymbol(typ),
+			},
+			pkgName,
+			typ,
```

Comment:
use keyed fields? otherwise it's hard to tell what's going on here. I personally always use keys when the number of fields is more than 2

Reply:
Fine, done.